### PR TITLE
first draft to a new MO: deploy django to aws

### PR DESCRIPTION
**Pull request by xavierlefevre**

### CODE REVIEW

**Review by tychota — CHANGES_REQUESTED** [review]
Hello @xavierlefevre, I fixed a bug I introduced working on dangerfiles: can you rebase https://github.com/bamlab/dev-standards/commit/6ebbc236cd6d51410519f78c03c62a10e48effc3

I can then help you reviewing.

(note that your file has to be .mo.md or .s.md)

File: backend/django/deploy-to-aws.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    5) eu-central-1 : EU (Frankfurt)
+    6) ap-southeast-1 : Asia Pacific (Singapore)
+    7) ap-southeast-2 : Asia Pacific (Sydney)
+    8) ap-northeast-1 : Asia Pacific (Tokyo)
+    9) sa-east-1 : South America (Sao Paulo)
+    10) cn-north-1 : China (Beijing)
+    (default is 3): 
+    ```
+  - Connect to your IAM user
+  - Give a general name to your instance, generaly the name of the project/company
```
Comment (Samox) [inline backend/django/deploy-to-aws.md]: example: name_of_project-staging

Comment (Samox) [reply backend/django/deploy-to-aws.md]: ah nan je retire c'est pas la ^^

File: backend/django/deploy-to-aws.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  - Enter your new application
+  - Go to configuration
+  - Modify the software configuration with variables such as:
+    ```
+    DJANGO_ACCOUNT_ALLOW_REGISTRATION, DJANGO_ADMIN_URL, DJANGO_ALLOWED_HOSTS, DJANGO_AWS_ACCESS_KEY_ID, DJANGO_AWS_SECRET_ACCESS_KEY, DJANGO_MAILGUN_API_KEY, DJANGO_SECRET_KEY, DJANGO_SECURE_SSL_REDIRECT, DJANGO_SENTRY_DSN, DJANGO_SERVER_EMAIL, DJANGO_SETTINGS_MODULE, MAILGUN_SENDER_DOMAIN, REDIS_ENDPOINT_ADDRESS, REDIS_PORT
+    ```
+- Check: You can verify on the AWS website that each server peace has been created (EB, RDS, S3, EC2)
+
+### Create a superadmin
+- TODO
```
Comment (Samox) [inline backend/django/deploy-to-aws.md]: Now that you have deployed once, on your host you should have the credentials for AWS (`cat ~/.aws/config`). 
To create a super user you will have to connect directly to the machine in order to run ./manage.py createsuperuser.

To ssh to your EC2 machine run `eb ssh [PUT YOUR INSTANCE NAME IF NOT BY DEFAULT]` 

Then on the machine go to the root of the project:
- `cd /opt/python/current`

Add the python environment:
- `source ./env`

Run the createuser command
- `cd /app`
- `./manage.py createsuperuser`

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/standard--django--deploy-to-aws/backend/django/deploy-to-aws.mo.md">deploy-to-aws.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by xavierlefevre** [comment]
@tychota ! rebased and renamed the file. @Samox reviewed the PR, what about merging it quickly, and iterating 💃

**Review by Samox — APPROVED** [review]

**Review by Samox — APPROVED** [review]

File: backend/django/deploy-to-aws.mo.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+## Owner: [Xavier Lefèvre](https://github.com/xavierlefevre)
+## Creation date: November, 13th 2017
+## Update date: November, 26th 2017
+
+## Context
+
+- You want to host your back-end with [Amazon Web Services](https://aws.amazon.com/)
+
+## Prerequisites
```
Comment (tychota) [inline backend/django/deploy-to-aws.mo.md]: Time

File: backend/django/deploy-to-aws.mo.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+- But you might still have errors `eb logs`, you are missing environment variables that you can directly add to AWS website
+  - On AWS website, on the top right, select the right server location (ex: London)
+  - Go to Elastic Beanstalk
+  - Enter your new application
+  - Go to configuration
+  - Modify the software configuration with variables such as:
+    ```
+    DJANGO_ACCOUNT_ALLOW_REGISTRATION, DJANGO_ADMIN_URL, DJANGO_ALLOWED_HOSTS, DJANGO_AWS_ACCESS_KEY_ID, DJANGO_AWS_SECRET_ACCESS_KEY, DJANGO_MAILGUN_API_KEY, DJANGO_SECRET_KEY, DJANGO_SECURE_SSL_REDIRECT, DJANGO_SENTRY_DSN, DJANGO_SERVER_EMAIL, DJANGO_SETTINGS_MODULE, MAILGUN_SENDER_DOMAIN, REDIS_ENDPOINT_ADDRESS, REDIS_PORT
+    ```
+- Check: You can verify on the AWS website that each server peace has been created (EB, RDS, S3, EC2)
```
Comment (tychota) [inline backend/django/deploy-to-aws.mo.md]: > Checks

File: backend/django/deploy-to-aws.mo.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+- Add the python environment:
+```bash
+source ./env
+```
+- Run the createuser command:
+```bash
+cd /app
+./manage.py createsuperuser
+```
+
```
Comment (tychota) [inline backend/django/deploy-to-aws.mo.md]: > Check you can access the admin

**Review by tychota — APPROVED** [review]

**Pull request merged**